compiler/src/iree/compiler/Codegen/TransformDialectStrategies/Common/AbstractReductionStrategy.h

```diff
@@ -34,8 +34,6 @@ struct AbstractReductionStrategy {
 
   virtual ~AbstractReductionStrategy() {}
```

Comment:
It looks like there's nothing virtual left so we can devirtualize the destructor.